### ISSUE TITLE
ERR: invalid transport header: encoding/hex: odd length hex string

### ISSUE THREAD

**Issue by eljefe80** [aler9/rtsp-simple-server#396]
## Which version are you using?

v0.16.1 from docker hub

## Which operating system are you using?

OS

- [ x] Linux
- [ ] Windows
- [ ] macOS

Architecture

- [ x] amd64
- [ ] arm64
- [ ] arm7
- [ ] arm6

## Describe the issue

I have a no-name ip camera that does not seem to be working with rtsp-simple-server. The error is as stated in the title: 

> ERR: invalid transport header: encoding/hex: odd length hex string

I have attached a tcpdump capture of the interchange between rtsp-simple server and this ip camera

## Describe how to replicate the issue

1. start the server

## Did you attach a network dump?

yes
[rtsp-simple-server-failure.txt](https://github.com/aler9/rtsp-simple-server/files/6525438/rtsp-simple-server-failure.txt)
[rtsp-simple-server-logs.txt](https://github.com/aler9/rtsp-simple-server/files/6525439/rtsp-simple-server-logs.txt)

**Comment by aler9** [comment]
Hello, thanks for reporting this issue and providing the logs, your camera is sending these kinds of (invalid) hex values:

```
ssrc=4317f
```

I can add support for your camera, but i need to understand if this value corresponds to `4317f0` or to `04317f`; in order to do this, i need a dump of the RTP packets between the camera and the server, that can be generated in this way:

1) Download wireshark (https://www.wireshark.org/)
2) Start capturing on the interface used for exchanging RTSP (the one of your network card)
3) Start reading the camera stream with VLC
4) Stop capturing after some seconds, save the result in .pcap format
5) Attach

**Comment by eljefe80** [comment]
Wireshark capture file attached.

Thanks!

[vlc-ipcam-dump.txt](https://github.com/aler9/rtsp-simple-server/files/6565012/vlc-ipcam-dump.txt)

**Comment by aler9** [comment]
Thanks, it seems that

```
4317f
```

is 

```
04317f
```

This is supported in main and will be added to the next release.

**Comment by eljefe80** [comment]
Hi, this seems to have moved us forward, but I'm getting a new error:

> ERR: invalid transport header: invalid SSRC

Should I open a new ticket for this?

Thanks!

**Comment by aler9** [comment]
I did another change, please try again with the latest main.

**Comment by koshkamau** [comment]
Hi!
I have a camera from "VOLDRELI 0322 Store" on Aliexpress. 
I tried the latest prebiult version of this rtsp-server and got an error message "[path proxied1] [rtsp source] ERR: invalid transport header: encoding/hex: odd length hex string".
Then I tried as suggested in the last message and everything works fine for now.

I just found out about this rtsp-simple-server but it looks very promising.
Thank you :)
